translate_math.js

```diff
@@ -0,0 +1,26 @@
+const katex = require("./katex/katex.js");
+const fs = require("fs");
+const escapeRegex = require("escape-string-regexp");
```

Comment:
If we wanted to remove a dependency, this one would be easy to get rid of - could just replace `left: "\\["` with `leftEscaped: "\\\\["`, etc.